docs/maintainers/release.md

```diff
@@ -0,0 +1,34 @@
+This file documents the list of steps to perform to create a new Antrea
+release. We use `<TAG>` as a placeholder for the release tag (e.g. `v0.1.0`).
+
+ * Create a release branch for each new minor release (e.g `release-0.1`). For
+   bug fixes, use the appropriate release branch.
+
+ * Open a PR against the appropriate release branch with the following commits:
+    1. a commit to update the [CHANGELOG](/CHANGELOG.md).
+    2. a commit to update [VERSION](/VERSION) as needed.
+
+ * Generate the deployment manifest for the release:
+   `IMG_NAME=antrea/antrea-ubuntu IMG_TAG=<TAG> ./hack/generate-manifest.sh --mode release > antrea.yml`
+
+ * Make the release on Github with the release branch as the target: copy the
+   relevant section of the [CHANGELOG](/CHANGELOG.md) for the release
+   description and upload the deployment manifest generated in the previous step
+   (as antrea.yml!). Check the `pre-release` box if applicable.
```

Comment:
just to confirm, `!` is by intention?

Reply:
yes it was but I was clarified my meaning by using a full sentence instead :)